instrumentation/redisson-3.0/redisson-3.0.gradle

```diff
@@ -15,5 +15,6 @@ muzzle {
 
 dependencies {
   library group: 'org.redisson', name: 'redisson', version: '3.0.0'
+  testImplementation group: 'org.redisson', name: 'redisson', version: '3.4.3'
```

Comment:
We intentionally use latest dep test to check both old versions and new version. I don't think we need to change the test dependency version for standard build.

Reply:
Yeah, we certainly want to test with the same version that we claim we support :) Meaning 3.0.0 in this case

Reply:
What about config the latestDepTestLibrary like ` latestDepTestLibrary group: 'org.redisson', name: 'redisson', version: '(,3.4.3)'`
because `redis:// ` is a break change.

Reply:
Oh shoot - so does that mean that redis:// doesn't work at all on the older versions? That's pretty unfortunate. I can't think of a simpler solution but the brute force is tweaking the Gradle build to set a property affecting the URL when testing latest deps. If this sounds too tricky, let's address it as a follow-up.

Reply:
Seems we can set the properties like this. 
```
test {
  if (testLatestDeps) {
    systemProperty "testLatestDeps", true
  }
}
```